vllm/engine/arg_utils.py

```diff
@@ -907,6 +908,9 @@ def create_engine_config(self, ) -> EngineConfig:
             embedding_mode=model_config.embedding_mode,
             preemption_mode=self.preemption_mode,
             num_scheduler_steps=self.num_scheduler_steps,
+            send_delta_data=(envs.VLLM_USE_RAY_SPMD_WORKER
+                             and envs.VLLM_SPMD_SEND_DELTA_DATA
```

Comment:
we should remove this env var. only `VLLM_USE_RAY_SPMD_WORKER` should be enough.

if you want to test and debug, you can directly change the code. there's no significance for adding it to the codebase.

Reply:
removed. cc @ruisearch42